common/src/TopJetIds.cxx

```diff
@@ -6,7 +6,7 @@ using namespace uhh2;
 
 bool CMSTopTag::operator()(const TopJet & topjet, const uhh2::Event &) const {
     auto subjets = topjet.subjets();
-    if(subjets.size() < 3) return false;
+    //if(subjets.size() < 3) return false;
```

Comment:
Thomas,

Good catch! I forgot I made that change. I shouldn't have committed it.

Doug

On Fri, Dec 18, 2015 at 2:35 PM, Thomas Peiffer <notifications@github.com>
wrote:

> In common/src/TopJetIds.cxx
> <https://github.com/UHH2/UHH2/pull/323#discussion_r48067162>:
>
> > @@ -6,7 +6,7 @@ using namespace uhh2;
> >
> >  bool CMSTopTag::operator()(const TopJet & topjet, const uhh2::Event &) const {
> >      auto subjets = topjet.subjets();
> > -    if(subjets.size() < 3) return false;
> > +    //if(subjets.size() < 3) return false;
>
> For the moment, I changed this line back to the original version with the
> subjets cut included.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UHH2/UHH2/pull/323/files#r48067162>.
>